dashboard/app/helpers/users_helper.rb

```diff
@@ -119,56 +107,73 @@ def user_summary(user)
     end
 
     unless exclude_level_progress
```

Comment:
Note to reviewers: This next section is much easier to read with GitHub's ignore whitespace setting, found under the "Diff settings" button at the top of your screen.  I recommend that setting for understanding the method extraction going on here.